src/components/Clock.tsx

```diff
@@ -17,15 +17,13 @@ const Clock = (props: {
 }) => {
   const [time, setTime] = useState<number>(props.maxtime);
   const refCircle = useRef<SVGCircleElement>(null);
-
-  const protectedTimeSound = useContext(DebateContext).conf.soundPack.pingProtectedTime;
-  const speechEndSound = useContext(DebateContext).conf.soundPack.pingSpeechEnd;
+  const conf = useContext(DebateContext).conf;
 
   const [audio1, state1, controls1] = useAudio({
-    src: protectedTimeSound,
+    src: conf.soundPack.pingProtectedTime,
   });
   const [audio2, state2, controls2] = useAudio({
-    src: speechEndSound,
+    src: conf.soundPack.pingSpeechEnd,
   });
```

Comment:
I changed the structure of the returned useAudio handles anywhere we use the library to reflect the way I implemented the change here: with an object to avoid name clutter.